sdk/search/search-documents/src/searchClient.ts

```diff
@@ -331,12 +331,19 @@ export class SearchClient<TModel extends object> implements IndexDocumentsClient
       select: this.convertSelect<TFields>(select) || "*",
       orderBy: this.convertOrderBy(orderBy),
       includeTotalResultCount: includeTotalCount,
-      vector: this.convertVector(vector),
+      vector: undefined,
+      vectors: undefined,
       answers: this.convertAnswers(answers),
       semanticErrorHandling: semanticErrorHandlingMode,
       debug: debugMode,
     };
 
+    if (vectors?.length === 1) {
+      fullOptions.vector = this.convertVector(vectors[0]);
+    } else {
+      fullOptions.vectors = vectors?.map(this.convertVector.bind(this));
+    }
```

Comment:
We should only use the `vectors` field here.  `vector` will be going away and the semantics are the same as a single value in `vectors`.

Reply:
Fixed in this followup PR #26777